acceptance/openstack/loadbalancer/v2/flavors_test.go

```diff
@@ -0,0 +1,33 @@
+//go:build acceptance || networking || loadbalancer || flavors
+// +build acceptance networking loadbalancer flavors
+
+package v2
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+	"github.com/gophercloud/gophercloud/acceptance/tools"
+	"github.com/gophercloud/gophercloud/openstack/loadbalancer/v2/flavors"
+)
+
+func TestFlavorsList(t *testing.T) {
```

Comment:
Same here, we should improve the acceptance tests.

You can take a look at https://github.com/gophercloud/gophercloud/blob/master/acceptance/openstack/loadbalancer/v2/loadbalancers_test.go for inspiration.

Reply:
I made a CRUD test